ARCHITECTURE.md

```diff
@@ -0,0 +1,56 @@
+# PrestoDB: Mission and Architecture
+
+## Introduction
+
+PrestoDB is an open-source distributed SQL query engine designed for fast analytic queries against data sources of all sizes, ranging from gigabytes to petabytes. It was originally developed by Facebook and later made available to the broader community. It is governed under the auspices of the Presto Foundation, a member of the Linux Foundation.
+
+## Mission
+
+### For users
+
+The primary mission of PrestoDB is to enable efficient and high-speed data processing for analytics and batch at scale. It aims to provide a single, unified query system that can access and process data stored in various formats and storage systems. Key aspects of its mission include:
+
+1. **High Performance:** Presto strives to exceed data warehouse performance in data lakes. To achieve this, Presto works to achieve vertical integration over modular portions of the stack.
+2. **Scalability:** Presto is used at extremely large data lakes across the world. As such, it aims for high degrees of scalability.
+3. **Connectivity and Integration:** Presto is designed to be flexible and adaptable to changes in infrastructure. This includes a highly customizable plugin infrastructure, including pluggable storage connectors.
+4. **User-Friendly:** Presto strives to be simple enough for a user to spin up and quickly test out, using familiar ANSI style SQL syntax, sensible default values and an optimizer that produces efficient query plans.
+
+### For developers
+
+Presto aims to accomplish the above goals for users by means of creating a broad, powerful, and collaborative open source community that strives for high standards in database engineering and design.
+
+1. **Open Source:** Develop and maintain as an open-source project, encouraging community contributions and collaboration.
+2. **Broad and open community:** Foster a large and active community to drive the project’s adoption and direction.
+3. **Composability**: Presto is designed for lakes and lakehouses. As such, it is engineered to fit cohesively in an ecosystem of other tools, and strives to integrate with industry standard specifications and libraries that help accomplish the user goals above.
+4. **Coding excellence**: Presto strives to maintain a high bar for contributions through good design, good abstractions, rigorous tests, and documentation.
+
+## Architecture
+
+Presto follows a distributed system model with a coordinator and multiple worker nodes. [See Presto Concepts for more information.](https://prestodb.io/docs/0.285.1/overview/concepts.html#server-types)
```

Comment:
can this link be version independent?

Reply:
https://prestodb.io/docs/current/overview/concepts.html#server-types